### DISCUSSION TITLE
About installation via Homebrew

### DISCUSSION THREAD

**Discussion by lewis-yeung** [JanDeDobbeleer/oh-my-posh#2023] [General]
I have recently tried to use an installation via Homebrew. I read the [documentation](https://ohmyposh.dev/docs/linux#installation) and noticed that it says an OMP executable will be installed to `$(brew --prefix)`. However, it seems that the executable is actually installed to `$(brew --prefix)/bin` due to [this](https://github.com/JanDeDobbeleer/homebrew-oh-my-posh/blob/295f24fa0f14d0fed43caadd97e01dc290fdc0da/oh-my-posh.rb#L15).

Should we correct it in the documentation?

I also noticed the following:

```bash
brew tap jandedobbeleer/oh-my-posh
brew install oh-my-posh
```

and a piece of description:

> If you want to use a standard theme, you can find them in `$(brew --prefix)/oh-my-posh/themes`, referencing them as such will always keep them compatible when updating Oh My Posh.

There is something interesting. After installing OMP by running the commands above, I found the themes installed in `$(brew --prefix)/share/oh-my-posh/themes` instead of `$(brew --prefix)/oh-my-posh/themes`. I felt confused but soon I realized that a formula of OMP is in **homebrew-core**, so `brew install oh-my-posh` installed OMP via **Homebrew/homebrew-core** instead of **jandedobbeleer/homebrew-oh-my-posh**, even though I used `brew tap jandedobbeleer/oh-my-posh`. It seems that is related to [this](https://github.com/Homebrew/homebrew-core/blob/e776f882cf9d183b0b57748f29b7327ee5338001/Formula/oh-my-posh.rb#L28).

Maybe we can just add another piece of description about this case to prevent other users from getting confused just like me.

If I use `brew install jandedobbeleer/oh-my-posh/oh-my-posh`, it will try to install OMP by building from source. However, this needs a Go installation via Homebrew as a dependency, ~~and unfortunately, I can't find a way to make it use my own `GOPROXY` env since the default (`proxy.golang.org`) is unavailable in my working environment.~~ The user-defined `GOPROXY` is now supported (JanDeDobbeleer/homebrew-oh-my-posh#9).

**Comment by JanDeDobbeleer** [comment]
I'll check but the only real one is the my repo. That one is under control, there's no proper automation for core. I'll adjust the docs.

**Reply by lewis-yeung** [reply]
Indeed. So happy to see OMP getting better and better.😆 Thank you!
